gluonnlp/data/conll.py

```diff
@@ -111,10 +112,11 @@ class CoNLL2000(_CoNLLSequenceTagging):
     ----------
     segment : {'train', 'test'}, default 'train'
         Dataset segment.
-    root : str, default '~/.mxnet/datasets/conll2000'
+    root : str, default '~/.mxnet/datasets/conll2000' or '$MXNET_HOME/datasets/conll2000'
```

Comment:
`root : str, default '$MXNET_HOME/datasets/conll2000'`, then add a line below to explain MXNET_HOME, such as
`$MXNET_HOME defaults to '~/.mxnet`.

Reply:
OK